modules/aws/elasticacheRedis/main.tf

```diff
@@ -1,9 +1,57 @@
-resource "aws_elasticache_cluster" "my_cluster" {
-  cluster_id           = var.name
-  engine               = "redis"
-  node_type            = var.node_type
-  num_cache_nodes      = var.num_cache_nodes
-  parameter_group_name = var.parameter_group_name
-  engine_version       = var.engine_version
-  port                 = 6379
-}
+module "redis" {
+  source = "github.com/cloudposse/terraform-aws-elasticache-redis.git"
```

Comment:
is cloudposse a better source for this module?

Reply:
that is the only source for the module. But one catch - need to add a specific version tag to this.

Reply:
Can you add a tag or commit id? that would be better

Reply:
done
can you approve so I can merge